bundles/best_practices.rst

```diff
@@ -75,7 +76,7 @@ configuration options (see below for some usage examples).
 Directory Structure
 -------------------
 
-The basic directory structure of an AcmeBlogBundle must read as follows:
+The basic directory structure of an ``AcmeBlogBundle`` must read as follows:
```

Comment:
I agree with @xabbuh here. It's a name and not a code-segment. Plus, any changes in text rendering is a minor break of the read flow, so having less in-line code blocks will improve docs readability.